examples/whisper.py

```diff
@@ -249,9 +245,7 @@ def listener(q):
 def init_whisper(model_name="tiny.en", batch_size=1):
   assert MODEL_URLS[model_name] is not None
 
-  filename = BASE / "whisper-{}.pt".format(model_name)
```

Comment:
The `whisper-` prefix was used during CI caching

Reply:
yeah good call. i'm definitely going to go through all of these again just to make sure I don't break any existing caching, etc.

Reply:
I updated the `workflow.yml` file to look at the new cache paths now.